### PR TITLE
added docs for ruuvitag

### PR DESCRIPTION
**Pull request by Alex9779**
## Description:
add docs for ruuvitag

**Related issue (if applicable):** fixes esphome/feature-requests#313

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#810

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Review by OttoWinter — APPROVED** [review]

File: components/sensor/ruuvitag.rst [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+- **acceleration_z** (*Optional*): The information for the acceleration z
+  sensor.
+
+  - **name** (**Required**, string): The name for the acceleration z sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **battery_voltage** (*Optional*): The information for the battery voltage
+  sensor
```
Comment (OttoWinter) [inline components/sensor/ruuvitag.rst]:
```suggestion
  sensor.
```

This should probably also include an additional sentence like "Only supported for RAWv2" protocol (people don't tend to read the top part from my experience)

Comment (Alex9779) [reply components/sensor/ruuvitag.rst]: I thought I mentioned it at the individual options...

File: components/sensor/ruuvitag.rst [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **tx_power** (*Optional*): The information for the transmit power
+  sensor
+
+  - **name** (**Required**, string): The name for the transmit power sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+  - Only available if RAWv2 protocol is used.
```
Comment (Alex9779) [inline components/sensor/ruuvitag.rst:130]: here, just available in RAWv2 and for all other options only with RAWv2 it is there too...

File: components/sensor/ruuvitag.rst [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **pressure** (*Optional*): The information for the pressure sensor.
+
+  - **name** (**Required**, string): The name for the pressure sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use
+    in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **acceleration_x** (*Optional*): The information for the acceleration
```
Comment (OttoWinter) [inline components/sensor/ruuvitag.rst]:
```suggestion
- **acceleration** (*Optional*): The information for the acceleration
```

I assume `acceleration` and `acceleration_x` are different things

**Review by OttoWinter — APPROVED** [review]

**Pull request merged**